src/common/types.cpp

```diff
@@ -892,16 +945,234 @@ idx_t StructType::GetChildCount(const LogicalType &type) {
 	return StructType::GetChildTypes(type).size();
 }
 
-LogicalType LogicalType::STRUCT(child_list_t<LogicalType> children) {
+LogicalType LogicalType::STRUCT(const child_list_t<LogicalType> &children) {
 	auto info = make_shared<StructTypeInfo>(move(children));
 	return LogicalType(LogicalTypeId::STRUCT, move(info));
 }
 
-LogicalType LogicalType::MAP(child_list_t<LogicalType> children) {
+LogicalType LogicalType::MAP(const child_list_t<LogicalType> &children) {
 	auto info = make_shared<StructTypeInfo>(move(children));
```

Comment:
Same here, this shouldn't be a const reference

Reply:
clang-tidy was not happy about that